src/collective/volto/formsupport/browser/send_mail_template_table.pt

```diff
@@ -0,0 +1,31 @@
+<tal:root xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+  i18n:domain="collective.volto.formsupport"
+  define="parameters python:options.get('parameters', {});
+            url python:options.get('url', '');
+            title python:options.get('title', '');">
+  <table>
+    <caption i18n:translate="send_mail_text">Email results for ${title}</caption>
```

Comment:
If i remember correctly, this should be write like this:

<caption i18n:translate="send_mail_text">Email results for <tal:title i18n:name="title">${title}</tal:title>

And then when you update locales, title will be dynamic